src/graphql/config.js

```diff
@@ -12,6 +12,11 @@ exports.config = {
     document: './queries/metadata.graphql',
     schema: { method: 'GET', url: 'https://metadata.p.rainbow.me/v1/graph' },
   },
+  simulation: {
```

Comment:
simulation needs POST requests this should only be used for simulation as POSTs mess with the cloudflare CDN